### DISCUSSION TITLE
Please add an integration for Microsoft Clarity

### DISCUSSION THREAD

**Discussion by phill579** [GitbookIO/community#198] [Feature Requests]
### What's your feature/idea?

Add integration for Microsoft Clarity

### What's your desired outcome?

To be able to review user recordings using Microsoft Clarity

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by addisonschultz** [comment]
Hey @phill579! 

Great idea. This is definitely something that can already be built with our Integration Platform! We're wrapping up a few things on our end, but it's already fully functional. You can find the docs for it here: https://developer.gitbook.com/

I've also added your request into our backlog for us to visit in the future!

**Reply by phill579** [reply]
Thanks @addisonschultz, how do I set up a developer account?

**Reply by addisonschultz** [reply]
@phill579  We have a guide for that here! 

https://developer.gitbook.com/getting-started/setup-guide
